app/assets/javascripts/discourse/app/helpers/application.js

```diff
@@ -50,3 +50,7 @@ registerUnbound("number", (orig, params) => {
 
   return htmlSafe(result);
 });
+
+registerUnbound("value-for-key", (obj = {}, key) => {
```

Comment:
I sent you a message about this, but I think this can be removed and we can use the `{{get}}` helper instead.